### ISSUE TITLE
sql: populate EXPLAIN of DELETE

### ISSUE THREAD

**Issue by dt** [cockroachdb/cockroach#4944]
currently:
```
 explain (debug) delete from deletes where k = 5;
 RowIdx | Key | Value | Disposition
--------+-----+-------+-------------
(0 rows)
```
Particularly since there may be fast or slow paths the query planner can take depending on the query (eg uses row values for filtering).

**Comment by knz** [comment]
Fixed as per 41c641afe03a7ef95d21fada87a7b78844e22106:

```
root@:26257> explain delete from foo where x = 5;
+-------+--------+--------------------------------+                                                                                                                                                         | Level |  Type  |          Description           |
+-------+--------+--------------------------------+                                                                                                                                                         |     0 | delete |                                |
|     1 | scan   | foo@blah /5-/5.000000000000001 |                                                                                                                                                         +-------+--------+--------------------------------+
```

**Comment by knz** [comment]
Woops, didn't see it's about DEBUG.
